platforms/documentation/docs/src/snippets/java/toolchain-management/kotlin/settings.gradle.kts

```diff
@@ -1,5 +1,5 @@
 plugins {
-    id("org.gradle.toolchains.foojay-resolver") version("0.8.0")
+    id("org.gradle.toolchains.foojay-resolver") version("0.9.0")
```

Comment:
🤔  This is a strange syntax: it's an infix call without a dot, yet the argument is in parenthesis